tests/test_isrFunctions.py

```diff
@@ -521,6 +522,98 @@ def test_ITLEdgeBleedMask(self):
+            3000: 3000 + detectorConfig.itlDipMinWidth + 1
+        ] |= satMaskValue
+
+        with self.assertLogs(level=logging.INFO) as cm:
+            ipIsr.isrFunctions.maskITLDip(exposure, detectorConfig)
+        self.assertEqual(len(cm[1]), 2)
+        self.assertIn("Found ITL dip (width 30; bkg 1000.00); masking column 992 to 1037", cm[1][0])
+        self.assertIn("Found ITL dip (width 16; bkg 1000.00); masking column 1495 to 1519", cm[1][1])
+
+        # This includes the scaled edges
+        np.testing.assert_array_equal(exposure.mask.array[:, 992: 1038] & dipMaskValue, dipMaskValue)
+        np.testing.assert_array_equal(exposure.mask.array[:, 991] & dipMaskValue, 0)
+        np.testing.assert_array_equal(exposure.mask.array[:, 1038] & dipMaskValue, 0)
+
+        np.testing.assert_array_equal(exposure.mask.array[:, 1495: 1520] & dipMaskValue, dipMaskValue)
+        np.testing.assert_array_equal(exposure.mask.array[:, 1494] & dipMaskValue, 0)
+        np.testing.assert_array_equal(exposure.mask.array[:, 1520] & dipMaskValue, 0)
+
+        # The other two should not be masked.
+        np.testing.assert_array_equal(exposure.mask.array[:, 2500] & dipMaskValue, 0)
+        np.testing.assert_array_equal(exposure.mask.array[:, 3000] & dipMaskValue, 0)
+
+        # Change the background to a much lower value, only 1 should be masked.
+        exposure.mask.array[:, :] &= ~dipMaskValue
+        exposure.image.array[:, :] = 50.0
+
+        with self.assertLogs(level=logging.INFO) as cm:
+            ipIsr.isrFunctions.maskITLDip(exposure, detectorConfig)
+        self.assertEqual(len(cm[1]), 1)
+        self.assertIn("Found ITL dip (width 30; bkg 50.00); masking column 992 to 1037", cm[1][0])
+
+        # This one should be masked.
+        np.testing.assert_array_equal(exposure.mask.array[:, 992: 1038] & dipMaskValue, dipMaskValue)
+        np.testing.assert_array_equal(exposure.mask.array[:, 991] & dipMaskValue, 0)
+        np.testing.assert_array_equal(exposure.mask.array[:, 1038] & dipMaskValue, 0)
+
+        # The other three should not be masked.
+        np.testing.assert_array_equal(exposure.mask.array[:, 1500] & dipMaskValue, 0)
+        np.testing.assert_array_equal(exposure.mask.array[:, 2500] & dipMaskValue, 0)
+        np.testing.assert_array_equal(exposure.mask.array[:, 3000] & dipMaskValue, 0)
+
+        # And blow things up so that masking is not applied.
+        # This needs to be in several sections because of the max width.
+        for i in range(12):
+            exposure.mask.array[:, i*100: i*100 + 50] |= satMaskValue
+        exposure.mask.array[:, 100: 100 + detectorConfig.itlDipMaxColsPerImage + 1] |= satMaskValue
+
+        maskBak = exposure.mask.array.copy()
+        with self.assertLogs(level=logging.WARNING) as cm:
+            ipIsr.isrFunctions.maskITLDip(exposure, detectorConfig)
+        self.assertEqual(len(cm[1]), 1)
+        self.assertIn("Too many (519) columns would be masked", cm[1][0])
```

Comment:
Have I confused myself in expecting that I would expect the number of columns to be masked to be at least 600?  You've added 12 sets of 50 SAT columns, so I don't see how this is coming up with 519 (as I would expect 600 to be the floor).  Maybe I'm just confused on lines 606-608.  Either way, can I ask for a short clarifying comment for this final test (lines 604 and onward)?